README.md

```diff
@@ -76,4 +76,4 @@ assert(val, { value: 5 });
 
 # License
 
-MIT / Apache-2.0
+[MIT](https://github.com/tauri-apps/tauri-plugin-store/blob/dev/LICENSE_MIT) / [Apache-2.0](https://github.com/tauri-apps/tauri-plugin-store/blob/dev/LICENSE_APACHE-2.0)
```

Comment:
I think you can use relative links, that'd be better in case we rename main branch/you preview a pr or whatever

Reply:
Good idea!